openapiv3/OpenAPIv3.proto

```diff
@@ -208,8 +208,8 @@ message Info {
   Contact contact = 4;
   License license = 5;
   string version = 6;
-  repeated NamedAny specification_extension = 7;
-  string summary = 8;
+  string summary = 7;
```

Comment:
This change was generated automatically after running the regeneration script. Is this expected?

Reply:
I've been manually changing this back to maintain backwards-compatibility of the protos. Could you do that here? Essentially I'm asking for the summary tag to be 7 and specification_extension should be 8.

Reply:
That would also require regenerating OpenAPIv3.pb.go

Reply:
Fixed. Updated both the .proto and .pb.go file.